### PR TITLE
Fix Issue#13 +event update and list will hang

### PR DESCRIPTION
**Pull request by jespirit**
Fixes https://github.com/pyrox18/SchedulerBot/issues/13 by checking if the event name is empty or null and displaying "N/A" as a replacement for embed information.

### CODE REVIEW

**Comment by pyrox18** [comment]
See #13 for updates.

File: SchedulerBot.Client/Parsers/EventParser.cs [hunk 1]
```diff
@@ -205,8 +205,9 @@ private static void ParseEventInputFlags(string[] args, ref Event evt, string ti
             }
         }
 
-        private static void ParseEventTimestamps(string bodyString, string timezone, ref Event evt)
+        private static void ParseEventTimestamps(string bodyString, string timezone, ref Event evt, bool update = false)
```
Comment (pyrox18) [inline SchedulerBot.Client/Parsers/EventParser.cs]: Can you change `update` to `isUpdate`? That makes it more obvious that it's a boolean.

File: SchedulerBot.Client/Parsers/EventParser.cs [hunk 2]
```diff
@@ -393,11 +394,45 @@ private static void ParseEventTimestamps(string bodyString, string timezone, ref
...
 
-                evt.Name = bodyString.RemoveCaseInsensitive(first.Text);
+                newBodyString = bodyString.RemoveCaseInsensitive(first.Text);
+
+                if (update)
+                {
+                    if (string.IsNullOrEmpty(newBodyString))
+                    {
+                        // This is fine. Don't update the event name
+                    }
```
Comment (pyrox18) [inline SchedulerBot.Client/Parsers/EventParser.cs]: Change this to the other form that you mentioned in the PR.

**Review by pyrox18 — CHANGES_REQUESTED** [review]

**Comment by jespirit** [comment]
Ok, I made the changes.

One thing that strikes me as odd is that I _may_ have messed up a rebase because I don't understand why your previous commits are being included in this pull request.

**Comment by pyrox18** [comment]
Might be because you targeted the master branch instead of the dev branch.

**Comment by jespirit** [comment]
Ahh, you're right. I noticed you changed it from Github. I didn't know you can do that.

**Review by pyrox18 — APPROVED** [review]

**Pull request merged**